test/Graphics/Transform.cpp

```diff
@@ -0,0 +1,353 @@
+            CHECK(transform.getMatrix()[0] == 3.14f);
+            CHECK(transform.getMatrix()[1] == 3.14f);
+            CHECK(transform.getMatrix()[2] == 0.0f);
+            CHECK(transform.getMatrix()[3] == 3.14f);
+            CHECK(transform.getMatrix()[4] == 3.14f);
+            CHECK(transform.getMatrix()[5] == 3.14f);
+            CHECK(transform.getMatrix()[6] == 0.0f);
+            CHECK(transform.getMatrix()[7] == 3.14f);
+            CHECK(transform.getMatrix()[8] == 0.0f);
+            CHECK(transform.getMatrix()[9] == 0.0f);
+            CHECK(transform.getMatrix()[10] == 1.0f);
+            CHECK(transform.getMatrix()[11] == 0.0f);
+            CHECK(transform.getMatrix()[12] == 3.14f);
+            CHECK(transform.getMatrix()[13] == 3.14f);
+            CHECK(transform.getMatrix()[14] == 0.0f);
+            CHECK(transform.getMatrix()[15] == 3.14f);
+        }
+    }
+
+    SUBCASE("Identity matrix")
+    {
+        CHECK(sf::Transform::Identity.getMatrix()[0] == 1.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[1] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[2] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[3] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[4] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[5] == 1.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[6] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[7] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[8] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[9] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[10] == 1.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[11] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[12] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[13] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[14] == 0.0f);
+        CHECK(sf::Transform::Identity.getMatrix()[15] == 1.0f);
+    }
+
+    SUBCASE("getInverse()")
+    {
+        SUBCASE("Identity matrix")
+        {
+            const auto inverse = sf::Transform::Identity.getInverse();
+            CHECK(inverse.getMatrix()[0] == 1.0f);
+            CHECK(inverse.getMatrix()[1] == 0.0f);
+            CHECK(inverse.getMatrix()[2] == 0.0f);
+            CHECK(inverse.getMatrix()[3] == 0.0f);
+            CHECK(inverse.getMatrix()[4] == 0.0f);
+            CHECK(inverse.getMatrix()[5] == 1.0f);
+            CHECK(inverse.getMatrix()[6] == 0.0f);
+            CHECK(inverse.getMatrix()[7] == 0.0f);
+            CHECK(inverse.getMatrix()[8] == 0.0f);
+            CHECK(inverse.getMatrix()[9] == 0.0f);
+            CHECK(inverse.getMatrix()[10] == 1.0f);
+            CHECK(inverse.getMatrix()[11] == 0.0f);
+            CHECK(inverse.getMatrix()[12] == 0.0f);
+            CHECK(inverse.getMatrix()[13] == 0.0f);
+            CHECK(inverse.getMatrix()[14] == 0.0f);
+            CHECK(inverse.getMatrix()[15] == 1.0f);
```

Comment:
You can just do:
```cpp
CHECK(inverse == sf::Transform::Identity);
```

After all, we already test `operator==` elsewhere, and this separates concerns.

Reply:
Fixed thanks.